x-pack/plugins/security_solution/docs/prebuilt_rules_customization_rfc.md

```diff
@@ -0,0 +1,2867 @@
+} 
+</pre>
+    </td>
+    <td><b>No</b></td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+} 
+</pre>
+    </td>
+    <td>N/A</td>
+    <td>
+      Prebuilt rule import rejected because no matching <br>
+       version found for existing rule_id security-asset
+      </td>
+    <tr>
+
+
+
+
+  <tr>
+    <td>
+      Not installed
+    </td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+  source_updated_at: "2024-05-..."
+} 
+</pre>
+    </td>
+    <td>Yes</td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+} 
+</pre>
+    </td>
+    <td>N/A</td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+  rule_source: {
+    type: "external",
+    is_customized: false,
+    source_updated_at: "2024-05-..."
+  }
+} 
+</pre>
+      </td>
+    <tr>
+
+
+
+  <tr>
+    <td>
+      Not installed
+    </td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+  source_updated_at: "2024-05-..."
+} 
+</pre>
+    </td>
+    <td>Yes</td>
+    <td>
+<pre>
+{
+  name: "My custom prebuilt rule",
+} 
+</pre>
+    </td>
+    <td>N/A</td>
+    <td>
+<pre>
+{
+  name: "My custom prebuilt rule",
+  rule_source: {
+    type: "external",
+    is_customized: true,
+    source_updated_at: "2024-05-..."
+  }
+} 
+</pre>
+      </td>
+    <tr>
+
+
+  <tr>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+  rule_source: {
+    type: "external",
+    is_customized: false,
+    source_updated_at: "2024-05-..."
+  }
+} 
+</pre>
+    </td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+  source_updated_at: "2024-05-..."
+} 
+</pre>
+    </td>
+    <td>Yes</td>
+    <td>
+<pre>
+{
+  name: "My custom prebuilt rule",
+} 
+</pre>
+    </td>
+    <td><b>No</b></td>
+    <td>
+    Rule import rejected<br> because of rule_id match
+      </td>
+    <tr>
+
+
+
+
+  <tr>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+  rule_source: {
+    type: "external",
+    is_customized: false,
+    source_updated_at: "2024-05-..."
+  }
+} 
+</pre>
+    </td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+  source_updated_at: "2024-05-..."
+} 
+</pre>
+    </td>
+    <td>Yes</td>
+    <td>
+<pre>
+{
+  name: "My custom prebuilt rule",
+} 
+</pre>
+    </td>
+    <td>Yes</td>
+    <td>
+<pre>
+{
+  name: "My custom prebuilt rule",
+  rule_source: {
+    type: "external",
+    is_customized: true,
+    source_updated_at: "2024-05-..."
+  }
+} 
+</pre>
+      </td>
+    <tr>
+
+
+
+
+
+
+  <tr>
+    <td>
+<pre>
+{
+  name: "My custom prebuilt rule",
+  rule_source: {
+    type: "external",
+    is_customized: true,
+    source_updated_at: "2024-05-..."
+  }
+} 
+</pre>
+    </td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+  source_updated_at: "2024-05-..."
+} 
+</pre>
+    </td>
+    <td>Yes</td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+} 
+</pre>
+    </td>
+    <td>Yes</td>
+    <td>
+<pre>
+{
+  name: "My prebuilt rule",
+  rule_source: {
+    type: "external",
+    is_customized: false,
+    source_updated_at: "2024-05-..."
+  }
+} 
+</pre>
+      </td>
+    <tr>
+
```

Comment:
It'd be nice to have a column named `Matches security-rule asses` to simpler reason about calculated `is_customized` value.